src/templates/email/confirm_account.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://ctfmgci.pythonanywhere.com/assets/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
We shouldn't hardcode these links, since the MGCI CTF Club website isn't associated with this project (even though it uses CTFOJ). Perhaps replace with the direct Bootstrap link (https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css)

Same goes for the style.css, we can probably make an inlined version with only the required styles

Reply:
_sigh_ picking between two terrible practices